test/asset/WebDriverAgentRunner-Runner.app/PkgInfo

```diff
@@ -0,0 +1 @@
+APPL????
```

Comment:
would it be necessary to regularly update WDA in the future? Can we instead add something that requires less maintenance?

Reply:
or can we build it on the fly from the corresponding npm module before running the tests?

Reply:
We shouldnt need to update it for newer xcode versions. This is not something that gets outdated